roles/openshift_aws/defaults/main.yml

```diff
@@ -254,22 +262,35 @@ openshift_aws_scale_group_health_check:
   type: EC2
 
 # these will be used during upgrade
-openshift_aws_master_group_config:
+openshift_aws_instance_config:
```

Comment:
We need build_ami and likely cns.  Who knows what other type will need to be added.  A little work here saves us a lot in the future.

Reply:
I only see one instance of 'openshift_aws_master_group_config' in this change.  This means it needs to be updated elsewhere to reflect the new variable name.

Also, it appears this doesn't need to have a child dictionary at all since it's most likely not going through the same tasks as it was previously (such as openshift_aws_node_group_config)

Definitely changing name to openshift_aws_master_instance_config makes good sense now.